data-raw/R2R_TMH_habitat_inputs/tmh_data_formatting.R

```diff
@@ -0,0 +1,206 @@
+library(tidyverse)
+
```

Comment:
might be helpful to have comments here on what the purpose of this script is.

Reply:
added a header